setup.py

```diff
@@ -89,6 +89,15 @@ def get_extensions():
     sources = main_file + source_cpu
```

Comment:
This is fine for now, but for a follow-up PR, can you remove from `glob` the files under `csrc/models`?

Reply:
@fmassa I don't exactly remember but I think that if you remove cpp files of models from extension setup won't compile and link to them. but I may be wrong.

Reply:
But we create a new namespace for C++ models, under `_C_test` or something like that, right? We don't need it under `_C`

Reply:
@fmassa this is the print of tests and sources in setup.py:
```
test files:
	 /home/shahriar/vision/test/test_models.cpp
	 /home/shahriar/vision/torchvision/csrc/models/vgg.cpp
	 /home/shahriar/vision/torchvision/csrc/models/resnet.cpp
	 /home/shahriar/vision/torchvision/csrc/models/mobilenet.cpp
	 /home/shahriar/vision/torchvision/csrc/models/shufflenetv2.cpp
	 /home/shahriar/vision/torchvision/csrc/models/densenet.cpp
	 /home/shahriar/vision/torchvision/csrc/models/alexnet.cpp
	 /home/shahriar/vision/torchvision/csrc/models/inception.cpp
	 /home/shahriar/vision/torchvision/csrc/models/googlenet.cpp
	 /home/shahriar/vision/torchvision/csrc/models/squeezenet.cpp

source files:
	 /home/shahriar/vision/torchvision/csrc/vision.cpp
	 /home/shahriar/vision/torchvision/csrc/cpu/ROIPool_cpu.cpp
	 /home/shahriar/vision/torchvision/csrc/cpu/ROIAlign_cpu.cpp
	 /home/shahriar/vision/torchvision/csrc/cpu/nms_cpu.cpp

```

I have separated the files from each other when I was resolving conflicts. Is this what you mean?

Reply:
Yes, that's exactly what I meant. Looks like you've done what I meant already, thanks!

Reply:
Great.